Tour-Collective-Devs/frontend-jobsite
# Js homepage landing

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #1 

## Description of Proposed Changes

## Steps to Test
1. Check to see that there is a placeholder splash image
1. Two links: Create an account && current openings
1. Brief description of why you should sign up

### Outline the steps to test
1. Checkout branch
1. Install any missing dependencies

```sh
git fetch --all
git checkout JS_homepage_landing
```

## Impacted Areas in Application

##### Modified

	modified:   package-lock.json
	modified:   package.json
	modified:   src/App.js
	modified:   src/index.js

##### New Files
	new file:   src/home/landingPage.js
	new file:   src/home/landingPage.css


List general components of the application that this PR will affect:

## Mentions @username

Tag users that need to review this code

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
1. Every class must be documented with purpose, author, and methods.
1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
